tensorflow/lite/examples/label_image/BUILD

```diff
@@ -35,11 +35,7 @@ cc_binary(
         "//tensorflow/lite/profiling:profiler",
         "//tensorflow/lite/tools/evaluation:utils",
         "@com_google_absl//absl/memory",
-    ] + select({
```

Comment:
Why are you removing the gpu delegate dependency ?

Reply:
Because `//tensorflow/lite/tools/evaluation:utils` includes gpu delegate dependency already. Those gpu delegate dependency lines were added before using  `//tensorflow/lite/tools/evaluation:utils`.

Reply:
Each build rule should keep the dependency on what it is directly uses. Relying on indirect dependency can make things break easily.
Please put the direct dependency
thanks

Reply:
OK, I'll do it.